integration/helpers/create-fixture.ts

```diff
@@ -18,7 +18,7 @@ interface FixtureInit {
   buildStdio?: Writable;
   sourcemap?: boolean;
   files: { [filename: string]: string };
-  template?: "cf-template" | "deno-template" | "node-template";
+  template?: "cf-template" | "deno-template" | "node-template" | "node-javascript-template";
```

Comment:
Not sure if we want an extra template for this though. 🤔

Maybe we should wait for the `convert-to-javascript` migration (see #3150) & call the migration before we do anything else in our tests?

Reply:
Hmmm... I think I'd prefer to duplicate than make a dependency here.

Reply:
This means we'll potentially have them double.

What about adding an extra (optional) key to `FixtureInit` that says if we want to use TypeScript or not.
If not, the `createFixture` function could run the migration automatically?
That way we don't need double templates + we can choose to use JS in our tests if we want to.

Reply:
I'm ok with that 👍